substrate/client/authority-discovery/src/worker.rs

```diff
@@ -677,9 +677,7 @@ where
 					metrics.dht_event_received.with_label_values(&["put_record_req"]).inc();
 				}
 			},
-			DhtEvent::StartProvidingFailed(..) => {},
-			DhtEvent::ProvidersFound(..) => {},
-			DhtEvent::ProvidersNotFound(..) => {},
+			_ => {},
```

Comment:
nit: Is this intended?

Reply:
Yes, as now we have much more events and we are only interested in a specific subset. So it doesn't make sense to list newly introduced events here.